logger/loggers.go

```diff
@@ -166,3 +166,10 @@ func displayError(err error) {
 		log.Printf("Logger write error: %v\n", err)
 	}
 }
+
+func disabled() bool {
+	if GlobalLogConfig.Enabled == nil {
+		return false
+	}
+	return !*GlobalLogConfig.Enabled
```

Comment:
Really minor, but would rather this become `enabled()` and invert that last check
Also, if the config is nil, then its not disabled?

Reply:
Yes I can change that logic around to check for an enabled